Sources/BraintreePayPal/BTPayPalVaultErrorHandlingEditRequest.swift

```diff
@@ -1,13 +1,15 @@
 import Foundation
 
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
 /// Options for the PayPal edit funding instrument flow with retry on failed attempts
 ///
 /// - Warning: This feature is currently in beta and may change or be removed in future releases.
-public struct BTPayPalVaultErrorHandlingEditRequest {
+public class BTPayPalVaultErrorHandlingEditRequest: BTPayPalVaultEditRequest {
 
-    private let editPayPalVaultID: String
-    private let riskCorrelationID: String
-    private let merchantAccountID: String?
+    public let riskCorrelationID: String
```

Comment:
I think we want to keep this private and have it only be set via the init

Reply:
With private, I run into issue when I access it in the dataCollector CMID function. 
I can make it internal I think